scripts/fixes/3810-broken-orders-with-tips.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Recreates transactions for successfull order based on the existing tip transactions.
+ * After that, remove broken tip transactions.
+ *
+ * Issue: https://github.com/opencollective/opencollective/issues/3810
+ */
+
+import { omit } from 'lodash';
+
+import * as constants from '../../server/constants/transactions';
+import { getHostFee, getPlatformFee } from '../../server/lib/payments';
+import { extractFees } from '../../server/lib/stripe';
+import models, { sequelize } from '../../server/models';
+
+const isDry = process.env.DRY;
+const affectedOrderIds = [
+  115250,
+  115238,
+  115105,
+  115104,
+  115065,
+  115064,
+  115047,
+  115045,
+  115044,
+  115021,
+  115000,
+  114969,
+  114952,
+  114751,
+  114750,
+  111824,
+  111822,
+  110638,
+  110640,
+];
```

Comment:
Hope no new one will appear before you trigger.

Reply:
I already deployed the patch.